src/core_modules/capture-core/components/DataStore/DataStore.epics.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import { ofType } from 'redux-observable';
+import { mergeMap, catchError } from 'rxjs/operators';
+import { EMPTY } from 'rxjs';
+import { fetchDataStore } from './DataStore.actions';
+import { getApi } from '../../d2';
+import type { UseNewDashboard } from './DataStore.actions';
+import { appStartActionTypes } from '../../../../components/AppStart';
+
+function getDataStoreFromApi() {
```

Comment:
Hey, 
Does it make sense to use the [DHIS2 hooks ](https://runtime.dhis2.nu/#/hooks/) to get/send the data from/to the API? 
Thanks!

Reply:
Hey,
I might be wrong, but since this is in an Epic I thought we couldn't use hooks? Since it's not a React function nor a component? Thanks!